docs/core/reference/cored.md

```diff
@@ -0,0 +1,95 @@
+# cored Command
+
+Command `cored` provides the Chain Core daemon and API server.
+
+`cored` is included with all official releases of Chain Core
+Developer Edition, and is run by default when starting Mac, Windows and Docker.
+
+If you plan to make changes to the internals of Chain Core, you can find
+build instructions in our [GitHub readme](https://github.com/chain/chain/blob/main/Readme.md#building-from-source);
+if you want to _use_ chain core, please download one of our official releases.
+
+### Build tags
```

Comment:
Sorta, but they are printed by Chain Core at startup, and in `cored -version`. I realize it's duplication from the README, but I think it's important for users to understand what the core is telling them without having to reference GitHub.